git_machete/cli.py

```diff
@@ -655,6 +655,9 @@ def strip_first_fragment(remote_branch: str) -> str:
             machete_client.read_definition_file()
             git.expect_no_operation_in_progress()
             current_branch = git.get_current_branch()
+            if "fork_point" in parsed_cli:
```

Comment:
Pls add some test case (at least for one of this commands) where an error is raised